browser/extensions/api/playlist/playlist_api.cc

```diff
@@ -0,0 +1,149 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/extensions/api/playlist/playlist_api.h"
+
+#include <memory>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "base/bind.h"
+#include "base/notreached.h"
+#include "brave/browser/playlist/playlist_service_factory.h"
+#include "brave/common/extensions/api/playlist.h"
+#include "brave/components/playlist/playlist_service.h"
+#include "brave/components/playlist/playlist_types.h"
+
+#if !defined(OS_ANDROID)
```

Comment:
this is unnecessary because extensions are not supported on android anyway

Reply:
actually I guess it's ok since desktop_playlist_player is not part of the api and is in an android guard